python/scylla/_rust/column_type.pyi

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Tuple
+class PyNativeType:
+    def __init__(self) -> None: ...
+
+class Int(PyNativeType): ...
+class Float(PyNativeType): ...
+class Double(PyNativeType): ...
+class Text(PyNativeType): ...
+class Boolean(PyNativeType):...
+class BigInt(PyNativeType):...
+
+class PyCollectionType:
+    frozen: bool
+
+
+class Map(PyCollectionType):
+    key_type: Any
+    value_type: Any
+
+
+class Set(PyCollectionType):
+    column_type: Any
+
+
+class List(PyCollectionType):
+    column_type: Any
+
+class PyTuple:
+    element_types: List[Any]
+
+class PyUserDefinedType:
+    name: str
+    frozen: bool
+    keyspace: str
+    field_types: List[Tuple[str, Any]]
```

Comment:
:wrench: I believe we should have an umbrella class for all CQL types, and use it instead of `Any` in all these places. Strong typing helps!